packages/angular/cli/README.md

```diff
@@ -106,15 +106,15 @@ You can find all possible blueprints in the table below:
 
 Scaffold  | Usage
 ---       | ---
-[Component](https://github.com/angular/angular-cli/wiki/generate-component) | `ng g component my-new-component`
-[Directive](https://github.com/angular/angular-cli/wiki/generate-directive) | `ng g directive my-new-directive`
-[Pipe](https://github.com/angular/angular-cli/wiki/generate-pipe)           | `ng g pipe my-new-pipe`
-[Service](https://github.com/angular/angular-cli/wiki/generate-service)     | `ng g service my-new-service`
-[Class](https://github.com/angular/angular-cli/wiki/generate-class)         | `ng g class my-new-class`
-[Guard](https://github.com/angular/angular-cli/wiki/generate-guard)         | `ng g guard my-new-guard`
-[Interface](https://github.com/angular/angular-cli/wiki/generate-interface) | `ng g interface my-new-interface`
-[Enum](https://github.com/angular/angular-cli/wiki/generate-enum)           | `ng g enum my-new-enum`
-[Module](https://github.com/angular/angular-cli/wiki/generate-module)       | `ng g module my-module`
+[Component](https://angular.io/cli/generate)      | `ng g component my-new-component`
```

Comment:
There's anchors to each generator individually, like https://angular.io/cli/generate#component. Can we use those instead here?

Reply:
Good one, done